pysages/backends/snapshot.py

```diff
@@ -2,25 +2,49 @@
 # Copyright (c) 2020-2021: PySAGES contributors
 # See LICENSE.md and CONTRIBUTORS.md at https://github.com/SSAGESLabs/PySAGES
 
+from jaxlib.xla_extension import DeviceArray as JaxArray
+from plum import dispatch
+from typing import NamedTuple, Union
+from pysages.utils import copy
 
 import jax.numpy as np
 
-from collections import namedtuple
 
-
-class Box(namedtuple("Box", ("H", "origin"))):
+class Box(NamedTuple("Box", [
+    ("H",      JaxArray),
```

Comment:
Is "H" really good name for this.
I haven't seen it elsewhere and could we use a better telling name?

Reply:
Agreed, I'm not sure what to call it, though. It represents the affine transform that needs to be applied to the positions for not orthogonal boxes. Maybe `transform` is good enough?

Reply:
`transform` sounds good to me

Reply:
OK. Will change that in a separate PR. Merging this for now.